solutions/standard_solutions/MediaOps/Tutorial_Scheduling_Encoder_Decoder.md

```diff
@@ -0,0 +1,142 @@
+
+    ![Edit Job Workflow](~/solutions/images/Scheduling_Edit_Job_Workflow.png)
+
+1. Open the `Add Node` panel and `+ Add Resource` **NewTek 001** from the `Encoder` resource pool to your job.
+
+    ![Add Encoder Resource](~/solutions/images/Scheduling_Edit_Job_Add_Encoder_Resource.png)
+
+1. Close the panel and make sure that you see your resource in the node-edge graph.
+
+    ![Check out the newly added resource](~/solutions/images/Scheduling_Workflow_Resource_Added.png)
+
+    > [!NOTE]  
+    > Depending on the Scheduling app **version**, you might need to hit the **refresh** button to see the newly added resource.
+
+## Step 3: Reserving the resource
+
+To reserve the resource for your job:
+
+1. Open the **Edit job** panel.
+
+1. `Save` the job `as Tentative`.
+
+    ![Save Job as Tentative](~/solutions/images/Scheduling_Edit_Job_Save_As_Tentative.png)
+    
+1. Close the `Edit Job` panel and confirm that your job's color has changed to **blue**, which indicated that it's in the `Tentative` state now.
+
+## Step 4: Add another resource
+
+Let's add another resource to our job, a `Decoder` this time.
+
+1. Open the `Edit job` panel and head over to the `Workflow` section.
+
+1. **Select** the `NewTek 001` resource and open the `Add After` panel.
+
+    ![Add After](~/solutions/images/Scheduling_Create_Job_Add_After.png)
+
+1. Select the **Decoder** pool. But there's no any resources available to add to our job. Worry not! We will take care of that. For now, to declare that we need a `Decoder` resource in our job, `Add Resource Pool` as a whole to the job.
+
+    ![Add Resource Pool](~/solutions/images/Scheduling_Create_Job_Add_Resource_Pool.png)
+
+    > [!TIP]  
+    > If you get an error in this step, make sure that you selected the resource before clicking `Add After` in the previous step.
+
+1. Resource pool should now be added to our job.
+
+    ![Resource Pool Added with Red Hand](~/solutions/images/Scheduling_Create_Job_Red_Hand.png)
+
+    > [!NOTE]
+    > Note the red hand icon on the resource pool in the `Nodes` table. It means that we still have to pick a concrete resource for our job.
+
+## Step 5: Swap resources
+
+Since the `NewTek 001` resource can act both as an `Encoder` and as a `Decoder`, let's swap it's role and use it as a `Decoder` in our job. Then we will still have the `Video Solutions 001` resource to be used as an `Encoder`. 
+
+1. Open the `Edit job` panel and head over to the `Nodes` table.
+
+1. Selece the `Encoder` row, and under the `Actions` column, click on the **swap action**.
+
+    ![Swap Action](~/solutions/images/Scheduling_Create_Job_Swap_Action.png)
+
+1. In the newly opened window, the `Encoder` resource pool should be preselected. Select the `Video Solutions 001` resource and click on the swap button.
+
+    ![Swap Resource](~/solutions/images/Scheduling_Create_Job_Swap_Resource.png)
+
+    > [!NOTE]  
```

Comment:
I would remove this note or point to the place where the refresh is located. Might be confusing that this is under the swap panel screenshot. And people refresh the browser, losing the panel they had open.